ci/get_labels/main.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+var prUrl = flag.String("url", "", "Url of pull request")
+
+type respLabels struct {
```

Comment:
Optional: What might be good is making a `type RespLabelCollection []RespLabel` then make that type implement the `Stringer` interface (i.e. make a `func (lc RespLabelCollection) String() string`). Then you can return a `RespLabelCollection` here and simply `fmt.Println(collection)`.